pkg/interceptors/tracing.go

```diff
@@ -0,0 +1,30 @@
+package interceptors
+
+import (
+	"google.golang.org/grpc"
+	grpctrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/google.golang.org/grpc"
+)
+
+// TracingUnaryServerInterceptor returns a unary server interceptor that will
+// trace requests to the given gRPC server.
+func TracingUnaryServerInterceptor(applicationName string) grpc.UnaryServerInterceptor {
+	return grpctrace.UnaryServerInterceptor(grpctrace.WithServiceName(applicationName))
+}
+
+// TracingStreamServerInterceptor returns a stream server interceptor that will
+// trace streaming requests to the given gRPC server.
+func TracingStreamServerInterceptor(applicationName string) grpc.StreamServerInterceptor {
+	return grpctrace.StreamServerInterceptor(grpctrace.WithServiceName(applicationName))
+}
+
+// TracingUnaryClientInterceptor returns a unary client interceptor that will
+// trace requests performed by gRPC client.
+func TracingUnaryClientInterceptor(applicationName string) grpc.UnaryClientInterceptor {
+	return grpctrace.UnaryClientInterceptor(grpctrace.WithServiceName(applicationName))
+}
+
+// TracingStreamClientInterceptor returns a stream server interceptor that will
+// trace streaming requests performed by gRPC client.
+func TracingStreamClientInterceptor(applicationName string) grpc.StreamClientInterceptor {
+	return grpctrace.StreamClientInterceptor(grpctrace.WithServiceName(applicationName))
+}
```

Comment:
We should consider adding a `grpc` suffix, similar to what we do for the ~`app`~ `http` tracer:

https://github.com/scribd/go-sdk/blob/702704e02320629af1acc1d62de799668ea3b741/pkg/instrumentation/router.go#L69-L70

Reply:
@fotos Added suffixes, please check